### ISSUE TITLE
Paper Mario Sticker Star slow with Vulkan on Android

### ISSUE THREAD

**Issue by TGP17** [citra-emu/citra#7220]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

All Citra builds with Vulkan support

### Description of Issue

Paper Mario Sticker Star is really slow with the Vulkan Graphics API on Android.
![Screenshot_20231129-215334_Citra](https://github.com/citra-emu/citra/assets/86961834/5f79dc6f-9cc9-4882-8d20-6da6761082ea)

### Expected Behavior

Paper Mario Sticker Star should run faster with Vulkan.

### Reproduction Steps

Download Citra on Android, make sure you're using the Vulkan Graphics API and start Paper Mario Sticker Star.

### Log File

[citra_log.txt](https://github.com/citra-emu/citra/files/13505461/citra_log.txt)

### System Configuration

CPU: Snapdragon 678
GPU/Driver: Adreno 612
RAM: 4GB
OS: Android 13

**Comment by mcbegamerxx954** [comment]
After trying this out by myself (in nightly 2726)
I can reproduce the issue, but for some reason setting the resolution to x2 temporarily resolves the issue even if you go back to 1x.

**Comment by TGP17** [comment]
You're right I can reproduce that switching resolution does fix it for the play session.

**Comment by mcbegamerxx954** [comment]
And as a last detail, underclocking the game after performing the x2 work around brings it to a nearly constant 100% speed for me even in x2

**Comment by TGP17** [comment]
Yeah I think that's probably due to FMV issues with this Game, so decreasing CPU Clock Speed helps in this case with performance

**Comment by TGP17** [comment]
This is also happening on he PC version, so it's not Android specific.
